recipes/libffi/all/conanfile.py

```diff
@@ -127,15 +127,12 @@ def generate(self):
             compile_wrapper = unix_path(self, os.path.join(self.source_folder, "msvcc.sh"))
             if architecture_flag:
                 compile_wrapper = f"{compile_wrapper} {architecture_flag}"
-            # FIXME: Use the conf once https://github.com/conan-io/conan-center-index/pull/12898 is merged
-            # env.define("AR", f"{unix_path(self, self.conf.get('tools.automake:ar-lib'))}")
             [version_major, version_minor, _] = self.dependencies.direct_build['automake'].ref.version.split(".", 2)
             automake_version = f"{version_major}.{version_minor}"
-            ar_wrapper = unix_path(self, os.path.join(self.dependencies.direct_build['automake'].cpp_info.resdirs[0], f"automake-{automake_version}", "ar-lib"))
             env.define("CC", f"{compile_wrapper}")
             env.define("CXX", f"{compile_wrapper}")
             env.define("LD", "link -nologo")
-            env.define("AR", f"{ar_wrapper} \"lib -nologo\"")
```

Comment:
automake doesn't yet define this conf, and the fix here is not correct. Not sure it's worth struggling c3i for something we know fails for sure now.